app/renderer/components/tabs/tabPage.js

```diff
@@ -32,23 +32,6 @@ class TabPage extends React.Component {
     this.onMouseLeave = this.onMouseLeave.bind(this)
```

Comment:
btw there's no listener for tab page previews anymore and I can't preview a tab set.